jupyter_book/utils.py

```diff
@@ -140,6 +140,12 @@ def _file_newer_than(path1, path2):
     return os.stat(path1).st_mtime > os.stat(path2).st_mtime
 
 
+def _minor_release(version):
+    """Excludes the patch release from a version number, such that
+    only major and minor versions face strict matching."""
+    return '.'.join(version.split('.')[:-1])
```

Comment:
I think a slightly more readable version of this could be:

```suggestion
    return version.rsplit('.', 1)[0])
```

so if we have a version like 0.6.1dev0, this should return only `0.6`

Reply:
I also added in a quick check to make sure that we keep adhering to the `major.minor.patch` version format. This is just a sanity check in case we forget about this in the future, does that work?

Reply:
The sanity check is a great call :+1:  Committing !